pixels-turbo/pixels-worker-vhive/README.md

```diff
@@ -2,7 +2,7 @@
 
 This module is for creating the Docker image for vHive cloud functions.
 
-Currently, you need to modify [pixels-trino/connector/pom.xml](https://github.com/pixelsdb/pixels-trino/blob/master/connector/pom.xml): replace the `pixels-invoker-lambda` with `pixels-invoker-vhive`.
+If you have not already done so when deploying Pixels, you need to modify [pixels-trino/connector/pom.xml](https://github.com/pixelsdb/pixels-trino/blob/master/connector/pom.xml): replace the `pixels-invoker-lambda` with `pixels-invoker-vhive`.
```

Comment:
Siyuan has added both pixels-invoker-lambda and pixels-invoker-vhive into pixels-trino/connector. We can remove this line from the README.